### PR TITLE
Testcontainers cloud learning path

### PR DESCRIPTION
**Pull request by sbenhoff007**
## Description

The Testcontainers Cloud learning path/guide explains what Testcontainers Cloud is, what it does, how to set it up, and how it works with CI/CD tooling. This topic was requested and it only covers Testcontainers Cloud by Docker, not the open-source Testcontainers library.

## Reviews

This PR is being submitted before the videos are edited and uploaded to YouTube. This PR will be updated when the videos are ready for publication.

- [ ] Technical review
- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

**Review by dvdksn — COMMENTED** [review]
in #21108, we have the videos at the top of the page, and the text below it. I can't see the videos for this PR yet, but for the SSSC/scout case, I think it works quite well. The videos introduce the concept, and the text below recaps/extends the topic covered in the video. We also do pretty much the same thing for the concept guides, but I realize we don't for the dbc/compose.

Should we maybe try and use a standardized format? I think the video on top works well for scout and the concepts.

**Comment by sbenhoff007** [comment]
> in #21108, we have the videos at the top of the page, and the text below it. I can't see the videos for this PR yet, but for the SSSC/scout case, I think it works quite well. The videos introduce the concept, and the text below recaps/extends the topic covered in the video. We also do pretty much the same thing for the concept guides, but I realize we don't for the dbc/compose.
> 
> Should we maybe try and use a standardized format? I think the video on top works well for scout and the concepts.

Yes, I agree we can move all of these to the top of the page.

File: content/guides/testcontainers-cloud/_index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - title: Running Testcontainers Tests Using GitHub Actions and Testcontainers Cloud
+      url: https://www.docker.com/blog/running-testcontainers-tests-using-github-actions/
+    - title: Testcontainers Cloud on the Docker Blog
+      url: https://www.docker.com/search/?_sf_s=testcontainers%20cloud
+---
+
+Testcontainers Cloud is a cloud-based solution designed to streamline and enhance the process of running integration tests using Testcontainers. Testcontainers is the open source framework, which  allows developers to easily spin up containerized dependencies such as databases, message brokers, and other services required for testing. By shifting the management of Testcontainers-based containers to the cloud, Testcontainers Cloud optimizes performance, reduces resource constraints on local machines or CI servers, and ensures consistent test environments. This solution is particularly beneficial for teams working on complex, distributed systems, as it allows for scalable, isolated, and reliable testing without the typical overhead of managing containers locally.
```
[the remark this answers is not in the document]
Comment (sbenhoff007) [reply content/guides/testcontainers-cloud/_index.md:30]: Agreed and updated.

**Comment by dvdksn** [comment]
Made a couple tiny edits and compressed the image.

**Review by dvdksn — APPROVED** [review]

**Pull request merged**